### ISSUE TITLE
NTR: Bile acids low in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8797]
New term request Bile acids low in urine (f363): Chenodeoxycholic  UMLS: 5542811

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000126
- label: Decreased urinary bile acid level
- parents: [Abnormal urine carboxylic acid level (HP:0031980)](https://hpo.jax.org/app/browse/term/HP:0031980)
- pmid: 
- def: The amount of bile acids in the urine, normalized for urine concentration, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8797
- synonyms:
